### PR TITLE
Create RBD keys on upgrade

### PR DESCRIPTION
**Pull request by leseb**

### CODE REVIEW

**Comment by leseb** [comment]
jenkins test pipeline

**Comment by leseb** [comment]
jenkins test pipeline

File: infrastructure-playbooks/rolling_update.yml [hunk 1]
```diff
@@ -197,6 +188,23 @@
...
+        dest: "/var/lib/ceph/{{ item.0 }}/"
+        caps:
+          mon: "allow profile {{ item.0 }}"
+        cluster: "{{ cluster }}"
+        containerized: "{{ 'docker exec ceph-mon-' + hostvars[mon_host]['ansible_hostname'] if containerized_deployment else None }}"
+      when:
+        - cephx
+      delegate_to: "{{ mon_host }}"
+      ignore_errors: True # this might fail for upgrade from J to L on rbd-mirror and also on partially updated clusters
+      with_together:
```
Comment (guits) [inline infrastructure-playbooks/rolling_update.yml]: I think what you want is a `with_nested` here

Comment (guits) [reply infrastructure-playbooks/rolling_update.yml]: fixed

**Comment by guits** [comment]
jenkins test pipeline

**Pull request merged**